taskchampion/xtask/src/main.rs

```diff
@@ -9,10 +9,16 @@ use std::io::Write;
 use std::path::PathBuf;
 
 pub fn main() -> anyhow::Result<()> {
-    let arg = env::args().nth(1);
-    match arg.as_deref() {
-        Some("codegen") => codegen(),
-        Some(arg) => anyhow::bail!("unknown xtask {}", arg),
+    let arguments: Vec<String> = env::args().collect();
+    if arguments.len() < 2 {
+        println!("Valid arguments are `codegen`, `msrv <arg1> <arg2>`");
+        return Ok(());
+    }
+
+    match arguments[1].as_str() {
+        "codegen" => codegen(),
+        "msrv" => msrv(arguments),
+        arg => anyhow::bail!("unknown xtask {}", arg),
         _ => anyhow::bail!("unknown xtask"),
```

Comment:
This `_` was to catch the `None` variant (`arg.as_deref()` is an `Option<..>`). Since there's no such variant anymore, the first three arms of this `match` capture all of the possibilities and this last one is not required.

Reply:
I changed my code a bit and now I believe these paths are reachable. Interestingly, the last two arms are helpful for providing feedback for two different types of failures. I left them in there for the time being but if the text output isn't useful, then I have no problem removing them.